PostHogExample/ContentView.swift

```diff
@@ -113,7 +113,7 @@ struct ContentView: View {
                         RepresentedExampleUIView()
                     }
 
-                    Text("Sensitive text!!").accessibilityIdentifier("ph-no-capture")
+                    Text("Sensitive text!!").postHogMask()
```

Comment:
I think we can clean all the accessibilityIdentifier in the swiftui samples since it will not work anyway, maybe replace all with posthogMask